### DISCUSSION TITLE
Can we set the destination of the Download to a custom folder (that is not the Browser's default Download folder)?

### DISCUSSION THREAD

**Discussion by ScottWilliamAnderson** [gildas-lormeau/SingleFile#970] [General]
I would like to save the HTML pages that the extension creates directly onto an external drive, as I have a lot of pages to download and not enough free space for all of them in the primary drive, where the default Download folder is located.

Is there a way that we could specify the directory where the files get downloaded? e.g. If the default download page is C:\User\Downloads - but I would like to save the files to D:\Backups\Webpages

I saw that we could create a custom folder in the default download folder for the extension to save to, but if my C: drive is too full to download all the pages I want to keep, rather than doing one at a time and moving them manually to the D: drive, could I specify to download them directly to the target folder?

Thank you, Gildas, much appreciated.

**Comment by gildas-lormeau** [comment]
It's possible but you have to install an additional program for that, see https://github.com/gildas-lormeau/single-file-companion-lite. Otherwise, you could create a symbolic link from the sub-folder in the Downloads folder, see https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/mklink.

**Comment by ScottWilliamAnderson** [comment]
Thank you!

**Comment by gildas-lormeau** [comment]
You're welcome
